assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationSetupCTAWidget.test.js

```diff
@@ -383,5 +415,117 @@ describe( 'AudienceSegmentationSetupCTAWidget', () => {
 
 			expect( queryByText( /Maybe later/i ) ).not.toBeInTheDocument();
 		} );
+
+		it( 'should call the enableAudienceGroups action when `Enable groups` CTA is clicked.', async () => {
+			const settings = {
+				configuredAudiences: null,
+				isAudienceSegmentationWidgetHidden: false,
+			};
+
+			// Set the data availability on page load to true.
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.receiveIsDataAvailableOnLoad( true );
+
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.receiveGetAudienceSettings( settings );
+
+			const { getByRole, queryByText } = render(
+				<AudienceSegmentationSetupCTAWidget Widget={ Widget } />,
+				{
+					registry,
+				}
+			);
+
+			expect(
+				getByRole( 'button', { name: /Enable groups/i } )
+			).toBeInTheDocument();
+
+			// eslint-disable-next-line require-await
+			await act( async () => {
+				fireEvent.click(
+					getByRole( 'button', { name: /Enable groups/i } )
+				);
+			} );
+
+			fetchMock.post( syncAvailableAudiencesEndpoint, {
+				status: 200,
+				body: audiencesFixture,
+			} );
+
+			const settingsBody = {
+				configuredAudiences: [
+					audiencesFixture[ 2 ].name,
+					audiencesFixture[ 3 ].name,
+				],
+				isAudienceSegmentationWidgetHidden: false,
+			};
+
+			fetchMock.post( audienceSettingsEndpoint, {
+				status: 200,
+				body: settingsBody,
+			} );
+
+			muteFetch( reportEndpoint );
+
+			expect( queryByText( 'Enabling groups' ) ).toBeInTheDocument();
+		} );
+
+		it( 'should call the enableAudienceGroups action when autoSubmit is set to true.', async () => {
```

Comment:
Please update the test description to remove the reference to `enableAudienceGroups`, for example:

```suggestion
		it( 'should initialise the list of configured audiences when autoSubmit is set to true.', async () => {
```